backend/models/core/Entity.js

```diff
@@ -228,7 +224,8 @@ class Entity {
               attributeType == "text" ||
               attributeType == "image" ||
               attributeType == "link" ||
-              attributeType === "video"
+              attributeType === "video" ||
+              attributeType === "date-time"
```

Comment:
The `datetime` attribute type should be saved in the `value_datetime` column and not on the `value_string` column. 

Also, can you please convert the text `date-time` to `datetime` instead?